src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/AbstractAddClassProjectCommand.cs

```diff
@@ -39,27 +39,9 @@ protected override Task<CommandStatusResult> GetCommandStatusAsync(IProjectTree
             }
         }
 
```

Comment:
Why do we still need this class? I expected it be merged with the "uber" class and follow the same pattern.